update.sh

```diff
@@ -303,7 +312,7 @@ print_expose_ports() {
 		expose_comment="Expose HTTP, HTTPS and Console ports"
 		expose_ports="8080 8443 8101"
 		;;
-	2.2.0|2.3.0|2.4.0-snapshot)
+	2.2.0|2.3.0|2.4.0-snapshot|2.4.0.M1|2.4.0.M2|2.4.0.M3)
```

Comment:
According to the [community post](https://community.openhab.org/t/openhab-2-4-milestone-builds/50359) the first usable milestone was M3. So I think noone should ever use the M1 and M2 milestone builds.

Reply:
You're right. I have removed M1 and M2.